SCons/Node/__init__.py

```diff
@@ -652,7 +659,7 @@ def get_executor(self, create: int=1) -> Executor:
             try:
                 act = self.builder.action
             except AttributeError:
-                executor = SCons.Executor.Null(targets=[self])  # type: ignore
+                executor = SCons.Executor.Null(targets=[self])  # type: ignore[assignment]
```

Comment:
This is an interesting case. While it's currently just suppressing the type-change for the Null executor, this feels like it *could* just return `None` instead. It'd be more in-line with how other classes (eg: Scanners) handle their optional arguments/returns. Left it alone because that'd obviously be a huge change, but it's something I've been thinking about